packages/manager/src/features/Kubernetes/NodePoolFirewallSelect.tsx

```diff
@@ -0,0 +1,78 @@
+import {
+  FormControlLabel,
+  Radio,
+  RadioGroup,
+  Stack,
+  Typography,
+} from '@linode/ui';
+import React from 'react';
+import { Controller, useFormContext } from 'react-hook-form';
+
+import { FirewallSelect } from '../Firewalls/components/FirewallSelect';
+
+import type { CreateNodePoolData } from '@linode/api-v4';
+
+export const NodePoolFirewallSelect = () => {
+  const { control, watch } = useFormContext<CreateNodePoolData>();
+  const watchedFirewallId = watch('firewall_id');
```

Comment:
```suggestion
  const { control } = useFormContext<CreateNodePoolData>();
  const watchedFirewallId = useWatch({ control, name: 'firewall_id' })
```

Can we use `useWatch` so we benfit from `react-hook-form`'s fine-grain reactivitly? 
(`watch('firewall_id')` causes the entire form to re-render when `firewall_id` changes whereas`useWatch` should isolate the re-render to only the component that are subscribed)